sdk/include/ndk/rtlfuncs.h

```diff
@@ -3986,6 +3986,10 @@ LdrRelocateImageWithBias(
     _In_ ULONG Invalid
 );
 
+ULONG
+NTAPI
+RtlGetExpWinVer(_In_ PVOID BaseAddress);
```

Comment:
Must be moved to a win32ss-specific user (ntuser.h or similar) header

Reply:
Moved.